### ISSUE TITLE
Problem installing meshcentral 1.1.6

### ISSUE THREAD

**Issue by baromi1984** [Ylianst/MeshCentral#5223]
When installing meshcentral, the server does not start. meshcentral version 1.1.6. Same behavior on Windows and Linux. When I try to start meshcentral it doesn't start completely and displays the following in the console:

`C:\Program Files\Open Source\MeshCentral\node_modules>node meshcentral
Installing node-windows@0.1.4...
Installing loadavg-windows...
Installing node-windows@0.1.4...
MeshCentral HTTP redirection server running on port 80.
Generating certificates, may take a few minutes...
Generating root certificate...
Generating HTTPS certificate...
Generating MeshAgent certificate...
Generating code signing certificate...
Generating Intel AMT MPS certificate...
MeshCentral v1.1.6, WAN mode.
Code signed MeshService64.exe.
Code signed MeshService.exe.
`
and nothing else happens.

Has anyone encountered such a problem? Please advise if there is any solution.

PS

This is a clean install with the default config.

**Comment by si458** [comment]
Does it close right after the last text?
Does it go back to the command line
As it won't do anything else

**Comment by baromi1984** [comment]
No, it doesn't close or return to the command line, it just stays in the same position. And still the same for more than an hour.

`C:\Program Files\Open Source\MeshCentral\node_modules>node meshcentral
Installing node-windows@0.1.4...
Installing loadavg-windows...
Installing node-windows@0.1.4...
MeshCentral HTTP redirection server running on port 80.
Generating certificates, may take a few minutes...
Generating root certificate...
Generating HTTPS certificate...
Generating MeshAgent certificate...
Generating code signing certificate...
Generating Intel AMT MPS certificate...
MeshCentral v1.1.6, WAN mode.
Code signed MeshService64.exe.
Code signed MeshService.exe.`

**Comment by si458** [comment]
That's correct, then if u visit ur ip address of the computer, you should get a web page

**Comment by baromi1984** [comment]
Тo, the web page does not open. Usually port 443 should up. But https server does not start.  I tried installing different versions of meshcentral. The only version that works is 0.9.99. Since version 1.0.0, the function of automatic code signing has appeared, and it seems to me that this is the problem, but so far I can’t figure out how to solve it.

**Comment by si458** [comment]
What version of node are you running? It needs to be at least version 18

**Comment by baromi1984** [comment]
I found a temporary solution. I commented out lines 3177-3196 and 248 in ./node_modules/meshcentral/meshcentral.js and the service started/
![изображение](https://github.com/Ylianst/MeshCentral/assets/100674210/e87dfd58-832e-48a5-b0dc-73aa2ace4058)

![изображение](https://github.com/Ylianst/MeshCentral/assets/100674210/613d0eb5-78ae-4e25-9677-9220c363969b)
